#
# ROSMIP Controller Debug
#

bool upright

float64 sp_theta
float64 sp_phi
float64 sp_phi_dot
float64 sp_gamma
float64 sp_gamma_dot

float64 wheel_angle_l
float64 wheel_angle_r
float64 theta
float64 phi
float64 gamma
float64 d1_u
float64 d2_u
float64 d3_u

float64 duty_l
float64 duty_r

